firebase_functions/functions/src/helpers.ts

```diff
@@ -79,246 +70,153 @@ const sanatiseObject = (device: any) => {
   return false;
 };
 
-const findTx = (hashes, provider, iotaApiVersion) => {
-  return new Promise((resolve, reject) => {
-    axios({
-      method: 'POST',
-      url: provider,
-      headers: {
-        'Content-Type': 'application/json',
-        'X-IOTA-API-Version': iotaApiVersion,
-      },
-      data: {
-        command: 'getTrytes',
-        hashes,
-      },
-    })
-      .then(response => {
-        const txBundle = response.data.trytes.map(trytes => asTransactionObject(trytes));
-        resolve(txBundle);
-      })
-      .catch(() => {
-        console.error(`findTx failed. Couldn't find your transaction`);
-        throw Error(`Couldn't find your transaction!`);
-        reject();
-      });
-  });
+/**
+ * Find message on tangle by messageId
+ * @param messageId of the message to find
+ * @returns message_metadata: https://client-lib.docs.iota.org/libraries/nodejs/api_reference.html#messagemetadata
+ */
+const findMessage = async (messageId: string): Promise<MessageMetadata> => {
+  const client = await getClient();
+  const message_metadata = await client.getMessage().metadata(messageId)
+  return message_metadata
 };
 
-const getBalance = async address => {
```

Comment:
do we still have a way to get a balance of the wallet?

Reply:
I removed it from helpers.ts because it wasn't used in index.ts anymore. Getting the balance is just a two liner. If there will be usage for such a function a the future I could readd it.
`  const client = await getClient();
  const balance = await client.getBalance(seed).get();`